cluster-autoscaler/provisioningrequest/apis/autoscaling.x-k8s.io/v1beta1/types.go

```diff
@@ -175,6 +175,9 @@ type Detail string
 // The following constants list all currently available Conditions Type values.
 // See: https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Condition
 const (
+	// Accepted indicates that the ProvisioningRequest was accepted by ClusterAutoscaler,
+	// so ClusterAutoscaler will attempt to provision the nodes for it.
+	Accepted string = "Accepted"
```

Comment:
Can we also add the `CapacityRevoked` condition?

Reply:
Added